### ISSUE TITLE
Add dye to mart!

### ISSUE THREAD

**Issue by ghost** [SmashMC-Development/Suggestions#744]
**Describe the suggestion**
Some of the materials for making different dyes are just too cumbersome to get and puts a stop on the things you are building. So, add dye to mart.

**Which servers does this apply to?**
All.

**Additional context**
N/A

**Comment by Dxhcky** [comment]
Yeah it's annoying to get ink sacs and other colours can be a pain to get too
